### PR TITLE
Update integration specs for copying dSYM

### PR DESCRIPTION
**Pull request by dnkoutso**
For https://github.com/CocoaPods/CocoaPods/pull/6536

### CODE REVIEW

File: install_add_pod/after/Pods/Target Support Files/Pods-SampleApp/Pods-SampleApp-frameworks.sh [hunk 1]
```diff
@@ -54,6 +54,15 @@ install_framework()
   fi
 }
 
+# Copies the dSYM of a vendored framework
+install_dsym() {
+  local source="$1"
+  if [ -r "$source" ]; then
+    echo "rsync -av --filter "- CVS/" --filter "- .svn/" --filter "- .git/" --filter "- .hg/" --filter "- Headers" --filter "- PrivateHeaders" --filter "- Modules" "${source}" "${DWARF_DSYM_FOLDER_PATH}""
```
Comment (segiddins) [inline install_add_pod/after/Pods/Target Support Files/Pods-SampleApp/Pods-SampleApp-frameworks.sh]: Quotes need escaping

Comment (dnkoutso) [inline install_add_pod/after/Pods/Target Support Files/Pods-SampleApp/Pods-SampleApp-frameworks.sh]: good catch fixing

**Comment by dnkoutso** [comment]
@segiddins any objections of merging this? 1.2.1 is out and I am hoping to bring this to 1.3.0 now

**Comment by dnkoutso** [comment]
Will ask for forgiveness instead! merging away.

**Pull request merged**